### DISCUSSION TITLE
Anyone able to help me to get Yourls fixed in time?

### DISCUSSION THREAD

**Discussion by voelspriet** [YOURLS/YOURLS#2914] [Q&A] [answered]
I am dependent on Yourls, I give courses to students and need the shortcuts. Upgrading the tool leads to the "SQLSTATE[01000]: Warning: 1265 Data truncated for column 'keyword' at row 578" error. Although solutions were posted, I don't get them, I don't know how to fix it. Is anyone prepared to fix it for me for an Amazon gift card? See below:

Upgrade YOURLS

Updating DB. Please wait...

Unable to update the DB.

You will have to manually fix things, sorry for the inconvenience :(

The errors were:

SQLSTATE[01000]: Warning: 1265 Data truncated for column 'keyword' at row 578

**Comment by ozh** [comment]
Hello, just saw your tweet as well. Please send me an email (ozh at ozh dot org) with details, I'll see if there is something I can do to help

**Comment by voelspriet** [comment, marked as answer]
Three coders worked long and hard on this, https://twitter.com/0xtosh found the solution. Thank you 
@0xtosh
 for finding the design error in 
@ozh
 yourls tool! The error was in conversion utf8 types.  The following fields were way too long and had to be deleted 
![image](https://user-images.githubusercontent.com/31323314/116716707-5b04bf80-a9d8-11eb-8e1d-d73aee46f517.png)

**Reply by voelspriet** [reply]
The tool was injected with 1000+ spam links btw

**Reply by ozh** [reply]
Then you probably have some serious security issues you should look into :)
